src/App.tsx

```diff
@@ -17,18 +32,29 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                todos={todosFromServer}
+                onFilterChange={setFilteredTodos}
```

Comment:
The method name `onFilterChange` does not follow the naming conventions for event handlers. A better name would be `handleFilterChange`. You can read more about it [here](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364).